### PR TITLE
Ingredient #2 Content update

### PR DESCRIPTION
**Pull request by Andromeda2284**
Added and altered CSS for improved paragraph looks on margin values that space out and ease page readability.

New content for the Ingredient's page.

### CODE REVIEW

File: src/main.tsx [hunk 1]
```diff
@@ -1,9 +1,10 @@
 import React from 'react'
-import ReactDOM from 'react-dom/client'
+import { createRoot } from 'react-dom/client'
 import App from './App.tsx'
-import './index.css'
 
-ReactDOM.createRoot(document.getElementById('root')!).render(
+const container: any = document.querySelector('#root');
```
Comment (Wancieho) [inline src/main.tsx:5]: what's this for? CSS?

File: src/App.css
Comment (Wancieho) [inline src/App.css:1]: I think we need to add CSS modules

File: src/App.tsx
Comment (Wancieho) [inline src/App.tsx:1]: Don't you want to generate dynamic content? Seems pointless having static data  in a React project

File: src/App.css [hunk 2]
```diff
@@ -226,15 +233,38 @@ h1 cite blockquote {
...
+  color: white;
+  text-decoration: underline;
+}
+
+a:hover {
+  color: cyan;
+  text-decoration: none;
 }
 
+#paragraph-content1 h1,
```
Comment (Wancieho) [inline src/App.css:252]: Why are the heading tags wrapped by elements with IDs?

File: src/App.css [hunk 3]
```diff
@@ -226,15 +233,38 @@ h1 cite blockquote {
...
+  margin-top: 50px;
   text-align: center;
 }
 
+ul li,
+ol li {
+  margin: 10px 0;
+}
 
+#paragraph-content1-wrapper {
```
Comment (Wancieho) [inline src/App.css:265]: We need to look at naming conventions

File: src/App.css [hunk 4]
```diff
@@ -202,12 +198,23 @@ h1 cite blockquote {
 
 }
 
+.Ingredient-benefits {
```
Comment (Wancieho) [inline src/App.css:201]: always lower case

Comment (Andromeda2284) [reply src/App.css:201]: ok

File: src/App.css (hunk 3 above)
Comment (Andromeda2284) [reply src/App.css:265]: Yeah I agree because I am not good at that

File: src/main.tsx (hunk 1 above)
Comment (Andromeda2284) [reply src/main.tsx:5]: I changed it to using React 18's API. It was complaining in the window console

File: src/App.tsx
Comment (Andromeda2284) [reply src/App.tsx:1]: I realized that this is static content and yes I want to but I am not at that rank yet. Once I am near I will let you know as I work with you after this mess is over these last 3months now going over.

**Comment by Andromeda2284** [comment]
Please merge it I have more content that needs to come. I applied a lot of changes so feel free to check up on my work next time on a call preferably asap, respectively.

File: src/App.css (hunk 2 above)
Comment (Andromeda2284) [reply src/App.css:252]: I changed most of the stuff so it should be better now. I am trying to track my progress but the next merge is over due and it slow me down. I cant merge it myself.

**Review by Wancieho — APPROVED** [review]

**Pull request merged**